src/page/ModifyMenu/index.tsx

```diff
@@ -103,8 +103,16 @@ export default function ModifyMenu() {
   };
 
   const confirmModifyMenu = () => {
-    modifyMenu();
-    goMyShop();
+    if (isComplete) {
+      if (!isMobile) {
+        openGoMyShopModal();
+        return;
+      }
+      modifyMenu();
+      goMyShop();
+    } else {
+      toggleConfirmClick();
+    }
```

Comment:
```suggestion
      goMyShop();
      return;
    }
      toggleConfirmClick();
    }
```
return 작성하면 else 안써도 될 거 같은데 어떻게 생각하시나요?

Reply:
굿~ 위에 이미 return 문이 있어서 맞춰주는게 좋을 것 같네요